README.md

```diff
@@ -31,6 +31,7 @@ USAGE
 # Commands
 <!-- commands -->
 * [`percy-agent exec`](#percy-agent-exec)
```

Comment:
Note that changed in this readme are all autogenerated.

Reply:
waat